catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java

```diff
@@ -332,6 +332,12 @@ public UpdateResponse update(UpdateRequest updateRequest) throws IngestException
         String attributeQuery = getQuery(attributeName, identifiers);
 
         SolrQuery query = new SolrQuery(attributeQuery);
+        // Set number of rows to the result size + 1.  The default row size in Solr is 10, so this
```

Comment:
Normally I would agree, but this old Solr code already has an obscene amount of inline comments